coordinator/src/main/java/com/tencent/rss/coordinator/CoordinatorGrpcService.java

```diff
@@ -135,7 +135,7 @@ public void heartbeat(
         .setRetMsg("")
         .setStatus(StatusCode.SUCCESS)
         .build();
-    LOG.debug("Got heartbeat from " + serverNode);
+    LOG.warn("Got heartbeat from " + serverNode);
```

Comment:
for 100+ shuffle server, there are too much heartbeat log

Reply:
Ok